confignode/src/test/resources/confignode2conf/iotdb-confignode.properties

```diff
@@ -20,7 +20,7 @@
 rpc_address=0.0.0.0
 rpc_port=22279
 consensus_port=22280
-target_confignode=0.0.0.0:22277
+target_confignode=0.0.0.0:22277:22278
```

Comment:
Even if we add consensus port, need to make this be consistent with confignode/src/assembly/resources/conf/iotdb-confignode.properties

Reply:
> 

fixed

Reply:
fixed